docs/spl-docs/time-modifiers.md

```diff
@@ -0,0 +1,177 @@
+
+```spl
+index=main earliest=-15s latest=now
+```
+
+Running the same search without including `latest=now` might produce unpredictable results.
+
+## Examples
+
+### Examples of Absolute time modifiers 
+
+1. **Using `earliest` for a Specific Start Date**
+   To search for events starting from January 1, 2023, you would use:
+   ```spl
+   ... | earliest=01/01/2023:00:00:00
+   ```
+   In this search, `latest` is not specified which means that it would default to the current time `now()`. This search retrieves all events from the beginning of January 1, 2023, onwards
+
+2. **Combining `earliest` and `latest` for a Specific Date Range**
+   To search for events during the first quarter of 2023, you would use:
+   ```spl
+   ... | earliest=01/01/2023:00:00:00 latest=03/31/2023:23:59:59
+   ```
+   This command captures all events from January 1, 2023, up to and including March 31, 2023, providing a complete view of the first quarter's data.
+
+### Examples of relative time modifiers
+
+For the following examples, the current time is Monday, 07 October 2024, 10:15:00 A.M.
+
+> **Note:** `24h` is usually, but not always, equivalent to `1d` due to Daylight Savings Time adjustments.
+
+| Time Modifier | Description | Resulting Time | Equivalent Modifiers |
+|---------------|-------------|----------------|----------------------|
+| `now` | Now, the current time | Monday, 07 October 2024, 10:15:00 A.M. | `now()` |
+| `-60m` | 60 minutes ago | Monday, 07 October 2024, 09:15:00 A.M. | `-60m@s` |
+| `-1h@h` | 1 hour ago, to the hour | Monday, 07 October 2024, 09:00:00 A.M. |  |
+| `-1d@d` | Yesterday | Sunday, 06 October 2024, 12:00:00 A.M. |  |
+| `-24h` | 24 hours ago (yesterday) | Sunday, 06 October 2024, 10:15:00 A.M. | `-24h@s` |
+| `-7d@d` | 7 days ago, 1 week ago today | Monday, 30 September 2024, 12:00:00 A.M. |  |
+| `-7d@m` | 7 days ago, snap to minute boundary | Monday, 30 September 2024, 10:15:00 A.M. |  |
+| `@w0` | Beginning of the current week | Sunday, 06 October 2024, 12:00:00 A.M. |  |
+| `+1d@d` | Tomorrow | Tuesday, 08 October 2024, 12:00:00 A.M. |  |
+| `+24h` | 24 hours from now, tomorrow | Tuesday, 08 October 2024, 10:15:00 A.M. | `+24h@s` |
+
+### Examples of searches with relative time modifiers
+
+#### Search from the Beginning of the Week to the Time of Your Search
+
+In this example, we're looking for events that occurred from the start of the current week up to the moment the search is executed.
+
+```spl
+... | earliest=@w0 latest=now
+```
+This query captures events from the very beginning of the week, starting at midnight on Sunday, all the way to the current time. For example, executing this search on a Wednesday at 3:00 P.M. would cover events over an approximate span of 87 hours.
+
+The equivalent time range for this query, if conducted at 3:00 P.M. on a Wednesday, extends from 00:00 A.M. on Sunday to 3:00 P.M. on Wednesday.
```

Comment:
nit: I think you can delete line 156 and the second sentence of line 154, because it doesn't really give the reader more relevant information; your description is already clear. What do you think?

Reply:
Yes, that makes sense and the description is self explanatory. I removed the two sentences.